devtools/gradle/gradle-application-plugin/src/test/java/io/quarkus/gradle/QuarkusPluginTest.java

```diff
@@ -1,10 +1,13 @@
 package io.quarkus.gradle;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
We don't use star imports in Quarkus, please disable them.

And given a few other things I see in the PR, I wanted to point you to https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#ide-config-and-code-style which might be useful if you plan to contribute regularly :).

Reply:
For some reason, the plugin does not pick up star static import. I updated the file.